index.js

```diff
@@ -96,14 +96,28 @@ const { parse, compileScript, compileTemplate } = require('@vue/compiler-sfc');
   }
 
   function getComponentsList(imports) {
+    if(!imports || !imports.length) {
+      return [];
+    }
+
     const dxComponents = Object.keys(imports).filter((cmp) => cmp.startsWith('Dx'));
     const appComponents = Object.keys(imports).filter((key) => imports[key].source.endsWith('.vue'));
     return [...dxComponents, ...appComponents];
   }
 
-  async function getCompositionApiSFC(source, template) {
-    const compiledScript = compileScript(parse(source).descriptor, {id: 'demo-'});
-    const compiledTemplate = compileTemplate({source: template ,id: 'demo-'});
+  function getCompositionApiSFC(source, template, convertTS = true) {
```

Comment:
Suggestion: Keep the default parameters for the top-level "translateSFC " function only, remove here (coz it may be not obvious for the caller).